### ISSUE TITLE
Saving notebook doesn't save plots in JupyterLab 1.0.1

### ISSUE THREAD

**Issue by dclong** [pyviz/pyviz_comms#44]
After the release of JupyterLab 1.0.0, there are some compatibility issues between holoviews and JupyterLab. When running a visualization code, the plot displays correctly in the notebook but saving the notebook doesn't save the plot. The plot is lost when reopening the notebook (after saving and closing it). 

Details:
OS: Ubuntu 18.10 (via Docker)
JupyterLab: 1.01   (installed using pip)
HoloViews: 1.12.3   (installed using pip)

**Comment by philippjfr** [comment]
Is it a problem in JLab 1.0.1 specifically? I can't reproduce this with 1.0.0 at least.

**Comment by dclong** [comment]
@philippjfr I'm so sorry about not testing the issue thoroughly. Just tried again and it seems to be a trust issue with JupyterLab. I'll close this issue and open a new one in the JupyterLab repository. 

1. When I created a brand new notebook and made a Holoviews visualization example there, everything works well. 

2. When I used a pre-saved notebook with a visualization example inside, the plot didn't show up after reopening the notebook. Manually trusting the notebook made the plot show up but it didn't persist, which means that I had to manually trust the notebook again next time I open the notebook. I don't have to go over the hassle if it's a newly created notebook.
